frontend/src/metabase/visualizations/lib/apply_tooltips.js

```diff
@@ -23,22 +23,26 @@ export function getClickHoverObject(
     settings,
   },
 ) {
-  let { cols } = series[0].data;
+  let { cols } = series[seriesIndex].data;
```

Comment:
I'm not sure why this was set to `0` before. I wondered if it was due to some data only being set on the first card in a breakout multi-series, but I didn't see anything to confirm that. Anyways, the new code makes more sense!

Reply:
Yes, I was not sure too. It caused showing wrong series name in the tooltip as described in this issue https://github.com/metabase/metabase/issues/15150

Thanks for the review 🙏